Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Junk4Dollars_appUITests.swift
+//  Junk4Dollars-appUITests
+//
+//  Created by John Riccardi on 10/28/19.
+//  Copyright © 2019 John Riccardi. All rights reserved.
+//
+
+import XCTest
+
+class Junk4Dollars_appUITests: XCTestCase {
+    func testHelloDefault() {
+        let app = XCUIApplication()
+        app.launch()
```

Comment:
How can you remove this duplication from each test?

Reply:
Moved the app launch to a setup function